packages/codemirror/src/pythonBuiltin.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { pythonLanguage } from '@codemirror/lang-python';
```

Comment:
This is a bit annoying as we try to only load language parser dynamically.

One way to achieve this would be to pass it at construction time using 

`ViewPlugin.define(view => new PythonBuiltin(view, pythonLanguage), { decorations: v => v.decorations })`

But first you should move the extension creation. Then this suggestion will be easier to implement.

Reply:
The import from `@codemirror/lang-python` is no longer needed after the extension is moved :+1:. It's just used to detect the current language